tests/functional/src/main/java/org/mule/tck/junit4/FunctionalTestCase.java

```diff
@@ -64,26 +66,49 @@ protected String getConfigResources()
         return null;
     }
 
-
     @Override
     protected ConfigurationBuilder getBuilder() throws Exception
     {
+        String domainConfig = getDomainConfig();
+        domainContext = domainConfig != null ? new DomainContextBuilder().setDomainConfig(domainConfig).build() : null;
+        SpringXmlConfigurationBuilder configurationBuilder;
         String configResources = getConfigResources();
         if (configResources != null)
         {
-            return new SpringXmlConfigurationBuilder(configResources);
+            configurationBuilder = new SpringXmlConfigurationBuilder(configResources);
         }
-        configResources = getConfigFile();
-        if (configResources != null)
+        else
         {
-            if (configResources.contains(","))
+            configResources = getConfigFile();
+            if (configResources != null)
+            {
+                if (configResources.contains(","))
```

Comment:
',' is a valid filename character, we shouldn't rely on this for deciding whether the string contains more than one filename.